include/vcpkg/base/messages.h

```diff
@@ -1043,6 +1043,11 @@ namespace vcpkg
                     "",
                     "Based on your system settings, the default path to store binaries is \"{path}\". This consults "
                     "%LOCALAPPDATA%/%APPDATA% on Windows and $XDG_CACHE_HOME or $HOME on other platforms.");
+    DECLARE_MESSAGE(DefaultTriplet,
+                    (msg::triplet),
+                    "",
+                    "Starting with the September 2023 release, the default triplet for builds will change from "
+                    "x86-windows to the detected host triplet ({triplet}).");
```

Comment:
Starting with the September 2023 release, the default triplet for builds will change from x86-windows to the detected host triplet ({triplet}). To resolve this message, add `--triplet {triplet}` for the new value, or `--triplet {value}` to keep the same behavior after the change.